build/build.py

```diff
@@ -206,7 +209,8 @@ def get_bazel_path(bazel_path_flag):
 def check_bazel_version(bazel_path):
   try:
     version_output = shell([bazel_path, "--bazelrc=/dev/null", "version"])
-  except subprocess.CalledProcessError:
+  except subprocess.CalledProcessError as e:
```

Comment:
Isn't this one redudant if you rethrow the exception above?

Reply:
Reverted